.gitignore

```diff
@@ -1009,3 +1009,6 @@ tools/unicode/c/genuca/x64
 tools/unicode/c/genuca/x86
 tools/unicode/c/icudefs.txt
 tools/unicodetools/com/ibm/rbm/lib
+
+# icu conversion tools (cldr-to-icu)
+tools/cldr/cldr-to-icu/target
```

Comment:
Please move this into `tools/cldr/cldr-to-icu/.gitignore` where it needs to be only `/target`

Reply:
Ahh thanks! Will do.